src/scanoss/Makefile

```diff
@@ -0,0 +1,58 @@
+######################################################################
+# Copyright (C) 2011 Hewlett-Packard Development Company, L.P.
```

Comment:
Please update the copyright information.

Reply:
Solved